server/src/test/java/io/crate/statistics/ColumnStatsTest.java

```diff
@@ -60,14 +58,28 @@ public void test_column_stats_generation() {
         assertThat(mostCommonValues.values().length, is(0));
     }
 
-    @Property
-    public void test_null_fraction_is_between_incl_0_and_incl_1(ArrayList<Integer> numbers,
-                                                                @InRange(minInt = 0) int nullCount,
-                                                                long totalRowCount) {
-        assumeThat(totalRowCount, greaterThanOrEqualTo((long) nullCount));
-        assumeThat((long) numbers.size() + nullCount, lessThanOrEqualTo(totalRowCount));
+    record Params(List<Integer> numbers, int nullCount, long totalRowCount){}
+
+    @Provide
+    Arbitrary<Params> arbitraryParams() {
```

Comment:
This looks less convenient than the previous solution. Is there something that's closer to what we had before?

Reply:
I know, in terms of code, it's more cumbersome, on the other hand I wanted to provide random but always valid params, before there were more runs where `assume` kicked in and the test didn't actually run, than the actual test runs.

- I saw at least one other library that supports junit5, so I will check it as well
- I can also revert this change here in the server module and keep pholser, since we don't have junit5 for the time being, and revisit it.